docs/api/paths/public/keyspace-_all_docs.yaml

```diff
@@ -9,7 +9,10 @@ parameters:
   - $ref: ../../components/parameters.yaml#/keyspace
 get:
   summary: Gets all the documents in the database with the given parameters
-  description: Returns all documents in the database based on the specified parameters.
+  description: |-
+    Returns all documents in the database based on the specified parameters.
```

Comment:
I think we should add the caveats and warnings about calling this endpoint here.

This document is also in `paths/admin/keyspace-all_docs.yaml`. Before we added some extra documentation I would have suggested deduplicating these yaml files, but we can leave them.